src/Microsoft.ML.StaticPipe/TransformsStatic.cs

```diff
@@ -1607,21 +1607,21 @@ private sealed class Reconciler : EstimatorReconciler
         /// speciﬁed shift-invariant kernel. With this transform, we are able to use linear methods (which are scalable) to approximate more complex kernel SVM models.
         /// </summary>
         /// <param name="input">The column to apply Random Fourier transfomration.</param>
-        /// <param name="dimension">The number of random Fourier features to create.</param>
+        /// <param name="rank">The number of random Fourier features to create.</param>
         /// <param name="useCosAndSinBases">If <see langword="true"/>, use both of cos and sin basis functions to create two features for every random Fourier frequency.
         /// Otherwise, only cos bases would be used.</param>
         /// <param name="generator">Which kernel to use. (if it is null, <see cref="GaussianKernel"/> is used.)</param>
         /// <param name="seed">The seed of the random number generator for generating the new features. If not specified global random would be used.</param>
-        public static Vector<float> LowerVectorSizeWithRandomFourierTransformation(this Vector<float> input,
-            int dimension = RandomFourierKernelMappingEstimator.Defaults.Rank, bool useCosAndSinBases = RandomFourierKernelMappingEstimator.Defaults.UseCosAndSinBases,
+        public static Vector<float> RandomFourierKernelMapper(this Vector<float> input,
```

Comment:
>RandomFourierKernelMapper [](start = 36, length = 25)

shall we go with RfKernelMappingEstimator

Reply:
Discussed with Artidoro. We feel `ApproximatedKernel` is a better name for describing this transform's behavior.

---
In reply to: [264837298](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264837298) [](ancestors = 264837298)